<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameForm</class>
 <widget class="QWidget" name="GameForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>110</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="2">
      <widget class="QPushButton" name="newRoundButton">
       <property name="toolTip">
        <string>Starts a new round of the game</string>
       </property>
       <property name="text">
        <string>New Round</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="computerScoreLabel">
       <property name="text">
        <string>Computer score</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QSpinBox" name="computerScoreSpinBox">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="layerScoreLabel">
       <property name="text">
        <string>Player's score</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QSpinBox" name="playerScoreSpinBox">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="playerMovesLabel">
       <property name="text">
        <string>Player moves</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="playerMovesSpinBox">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="playerMissesLabel">
       <property name="text">
        <string>Player misses</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QSpinBox" name="playerMissesSpinBox">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="playerHitsLabel">
       <property name="text">
        <string>Player hits</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QSpinBox" name="playerHitsSpinBox">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QLabel" name="playerDeadLabel">
       <property name="text">
        <string>Player dead</string>
       </property>
      </widget>
     </item>
     <item row="1" column="7">
      <widget class="QSpinBox" name="playerDeadSpinBox">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="computerMovesLabel">
       <property name="text">
        <string>Computer moves</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="computerMovesSpinBox">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="computerMissesLabel">
       <property name="text">
        <string>Computer misses</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QSpinBox" name="computerMissesSpinBox">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QLabel" name="computerHitsLabel">
       <property name="text">
        <string>Computer hits</string>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QSpinBox" name="computerHitsSpinBox">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="6">
      <widget class="QLabel" name="computerDeadLabel">
       <property name="text">
        <string>Computer dead</string>
       </property>
      </widget>
     </item>
     <item row="2" column="7">
      <widget class="QSpinBox" name="computerDeadSpinBox">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder></zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
